quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,514 @@
+        Atoms object.
+    cores
+        Number of cores to use for the Q-Chem calculation.
+        Effectively defaults to use all cores available on a given node, so this only needs to
+        be set by the user if less than all available cores should be used.
+    charge
+        The total charge of the molecular system.
+        Effectively defaults to zero.
+    mult
+        The spin multiplicity of the molecular system.
+        Effectively defaults to the lowest spin state given the molecular structure and charge.
+    xc
+        Exchange-correlation functional.
+        Defaults to wB97M-V.
+    basis
+        Basis set.
+        Defaults to def2-TZVPD.
+    scf_algorithm
+        Algorithm used to converge the SCF.
+        Defaults to DIIS, but for particularly difficult cases, GDM should be employed instead.
+    pcm_dielectric
+        Dielectric constant of the optional polarizable continuum impicit solvation model.
+        Defaults no None, in which case PCM will not be employed.
+    smd_solvent
+        Solvent to use for SMD implicit solvation model. Examples include "water", "ethanol", "methanol",
+        and "acetonitrile". Refer to the Q-Chem manual for a complete list of solvents available.
+        Defulats to None, in which case SMD will not be employed.
+    swaps
+        Dictionary of custom kwargs for the calculator. Must be formatted consistently with Pymatgen's
+        QChemDictSet's overwrite_inputs. For example: {"rem": {"symmetry": "true"}}.
+
+    Returns
+    -------
+    dict
+        Dictionary of results from quacc.schemas.ase.summarize_run
+    """
+
+    if pcm_dielectric is not None and smd_solvent is not None:
+        raise RuntimeError("PCM and SMD cannot be employed simultaneously! Exiting...")
+
+    true_charge, true_spin = true_charge_and_spin(atoms, charge, mult)
+
+    input_atoms = deepcopy(atoms)
+
+    overwrite_inputs = {"rem": {"method": xc}}
+
+    swaps = swaps or {}
+    for key in swaps:
+        if key not in overwrite_inputs:
+            overwrite_inputs[key] = {}
+            for subkey in swaps[key]:
+                overwrite_inputs[key][subkey] = swaps[key][subkey]
+
+    qchem_input_params = {
+        "basis_set": basis,
+        "scf_algorithm": scf_algorithm,
+        "qchem_version": 6,
+        "pcm_dielectric": pcm_dielectric,
+        "smd_solvent": smd_solvent,
+        "overwrite_inputs": overwrite_inputs,
+    }
+
+    if scf_algorithm.lower() == "gdm":
+        qchem_input_params["max_scf_cycles"] = 200
+
+    calc = QChem(
+        input_atoms=atoms,
+        cores=cores,
+        charge=charge,
+        spin_multiplicity=mult,
+        qchem_input_params=qchem_input_params,
+    )
+    atoms.calc = calc
+    atoms = run_calc(atoms)
+    return summarize_run(
+        atoms,
+        input_atoms=input_atoms,
+        additional_fields={
+            "name": "Q-Chem Static",
+            "charge": true_charge,
+            "spin_multiplicity": true_spin
```

Comment:
Can remove these now. `summarize_run` and related functions now take a `charge_and_mult: Tuple[int.int]` keyword.